polyfills/Element/prototype/childElementCount/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "browsers": {
+    "firefox": "<3.5",
+    "ie": "6 - 8",
+    "opera": "<10.0",
+    "safari": "<4"
```

Comment:
Can you add targeting for the remaining browser families here?  Does the polyfill work on iOS or Android, for example?  Blackberry, samsung?

[also applies to all the other configs included in this PR]